### DISCUSSION TITLE
Github Education Benefits Still Not Avaible to me

### DISCUSSION THREAD

**Discussion by HannahStarzProductions** [community/community#142616] [GitHub Education]
### Hi everyone,

Hi yall, i recently got approved for Github Student/Education but i have not gotten my access to the benifits and im a bit worried as its been a few days. Is it possible to know when im supposed to get it or whats holding it up or something like that. Im just really excited to get into the stuff

**Comment by Aninooo** [comment]
Go to the [GitHub Education benefits page](https://education.github.com/pack) and confirm that your pack is listed as “Active.” Sometimes the activation page can be delayed, so double-checking here can help.

**Comment by Jxstin1337** [comment]
Hi Hannah,

as long as it doesn't take longer than 2 weeks, everything should be fine.
Otherwise you could check your approval statuses here as well to be safe:
- [GitHub Education Approval Status](https://education.github.com/discount_requests/application)
- [GitHub Education Student Dev Pack](https://education.github.com/learner/learn)

**Comment by davevad93** [comment]
Hi @HannahStarzProductions ,

Don't worry, the waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @HannahStarzProductions was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
